src/app.rs

```diff
@@ -615,6 +615,17 @@ pub fn build_app() -> App<'static, 'static> {
                          argument. Changes the usage to `fd [FLAGS/OPTIONS] --search-path <path> \
                          --search-path <path2> [<pattern>]`",
                 ),
+        )
+        .arg(
+            Arg::with_name("strip-prefix")
+                .long("strip-prefix")
```

Comment:
How about a rename to `--strip-cwd-prefix`? Maybe that would make it a bit clearer what kind of prefix will be fixed.